x-pack/plugins/infra/public/components/logging/log_text_stream/text_styles.tsx

```diff
@@ -10,6 +10,8 @@ import React, { useMemo, useState, useCallback } from 'react';
 import { euiStyled, css } from '../../../../../observability/public';
 import { TextScale } from '../../../../common/log_text_scale';
 
+export type WrapMode = 'none' | 'original' | 'long';
```

Comment:
Nit: Is there a better name we could use than `original`, it doesn't really represent the *behaviour* of that wrap mode (like `long` does). (I appreciate this means changing quite a lot of references, so feel free to ignore).

Reply:
Would `source` be more descriptive? Or `pre-wrapped`?

Reply:
`pre-wrapped` is good and suggests the behaviour to me 👍

Reply:
ok, done